src/OpalCompiler-Tests/OCEnvironmentScopeTest.class.st

```diff
@@ -10,22 +10,26 @@ Class {
 { #category : #'tests - environment' }
 OCEnvironmentScopeTest >> testCompileWithEnvironment [
 	"testing compiling with environments other then Smalltalk globals. Note: the class in the environment has to point to the environment. And we have to compile for a class in the environment"
-
-
 	| environment method return | 
 	environment := SystemDictionary new.
 	environment at: #MyClass put: Point copy.
-	(environment at: #MyClass) environment: environment.
+	environment at: #MyClass2 put: Association.
```

Comment:
This test demonstrates the environment override. In the current image it does not work.